src-tauri/src/main.rs

```diff
@@ -117,6 +118,16 @@ fn is_debug_mode() -> bool {
     return cfg!(debug_assertions);
 }
 
+#[tauri::command]
+/// Returns true if linux compatible
+fn linux_checks() -> bool {
+    if get_host_os() == "windows" { 
+        false
+    } else {
+        linux_checks_librs()
+    }
+}
```

Comment:
Should probably be changed to returning a `Result<T,E>` type in the future but fine as is for now for me ^^